packages/design/src/colors.css

```diff
@@ -58,35 +58,39 @@
   --color-overlay--dimmed: rgba(var(--color-white--rgb), 0.6);
 
   /* Brand */
-  --color-brand: var(--color-green);
+  --color-brand: rgb(124, 175, 14);
   --color-brand--highlight: rgb(132, 234, 0);
 
   /* Jobber Workflow Colors */
-  --color-request: var(--color-indigo);
-  --color-quote: var(--color-pink);
-  --color-job: var(--color-yellowGreen);
-  --color-visit: var(--color-green);
+  --color-request: rgb(185, 133, 70);
+  --color-quote: rgb(164, 70, 88);
+  --color-job: var(--color-green);
+  --color-visit: var(--color-job);
   --color-task: var(--color-navy);
   --color-event: var(--color-yellow);
-  --color-invoice: var(--color-purple);
-  --color-payments: var(--color-orange);
+  --color-invoice: rgb(49, 98, 144);
+  --color-payments: var(--color-invoice);
 
-  --color-client: var(--color-teal);
+  --color-client: var(--color-blue);
 
   /* Jobber Base Palette */
-  --color-blue: rgb(1, 41, 57);
-  --color-blue--light: rgb(77, 105, 116);
-  --color-blue--lighter: rgb(153, 169, 176);
-  --color-blue--lightest: rgb(217, 223, 225);
-  --color-blue--dark: rgb(1, 27, 37);
-
-  --color-green: rgb(125, 176, 14);
-  --color-green--light: rgb(164, 200, 86);
-  --color-green--lighter: rgb(203, 223, 159);
-  --color-green--lightest: rgb(236, 243, 219);
-  --color-green--dark: rgb(81, 114, 9);
-
-  --color-lime: rgb(153, 193, 70);
+  --color-blue: rgb(1, 40, 55);
+  --color-blue--light: rgb(78, 105, 116);
+  --color-blue--lighter: rgb(131, 151, 160);
+  --color-blue--lightest: rgb(218, 224, 226);
+  --color-blue--dark: rgb(1, 25, 35);
+
+  --color-taupe: rgb(241, 239, 233);
```

Comment:
@fagnanm can you confirm these match the latest values you're using? Also relevant in the semantic section where I know "background" has been through some tuning.

Reply:
`color-taupe--light` in web is quite different (mobile is the same as what you have here) 

![Screenshot 2024-04-09 at 12 44 50 PM](https://github.com/GetJobber/atlantis/assets/104797704/60f98b64-a794-41c1-975a-223a0e908d80)

Reply:
OK, gotcha - I just pulled latest master and I'm getting the same values you have here, so I'll update that. Good catch!

Reply:
[Updated](https://github.com/GetJobber/atlantis/pull/1839/commits/78e4a433a7927b6f6b48ba3545757e234fd1a4ec)